src/Akeneo/Category/back/Infrastructure/Job/CleanCategoryTemplateEnrichedValuesProvider.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Category\Infrastructure\Job;
+
+use Akeneo\Tool\Component\Batch\Job\JobInterface;
+use Akeneo\Tool\Component\Batch\Job\JobParameters\ConstraintCollectionProviderInterface;
+use Akeneo\Tool\Component\Batch\Job\JobParameters\DefaultValuesProviderInterface;
+use Symfony\Component\Validator\Constraints\Collection;
+use Symfony\Component\Validator\Constraints\Type;
+
+/**
+ * @copyright 2023 Akeneo SAS (https://www.akeneo.com)
+ * @license   https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class CleanCategoryTemplateEnrichedValuesProvider implements ConstraintCollectionProviderInterface, DefaultValuesProviderInterface
+{
+    private const JOB_NAME = 'clean_category_template_and_enriched_values';
```

Comment:
You can inject the job name using the parameter `%pim_category.job_name.clean_category_template_and_enriched_values%` too
```suggestion
    public function __construct(readonly string $jobName) {}
```